packages/nextjs/app/builders/0x6BF7d6b94282BD48ff458599aDafA268BcB009FF/page.tsx

```diff
@@ -0,0 +1,144 @@
+import Image from "next/image";
+import { IconType } from "react-icons";
+import { FaEnvelope, FaGithub, FaLinkedin, FaMapMarkerAlt, FaTwitter } from "react-icons/fa";
+
+interface ProfileData {
+  name: string;
+  title: string;
+  bio: string;
+  avatar: string;
+  location: string;
+  email: string;
+  social: {
+    github: string;
+    twitter: string;
+    linkedin: string;
+  };
+}
+
+interface StyledIconProps extends React.HTMLAttributes<HTMLSpanElement> {
+  icon: IconType;
+  size?: number;
+  color?: string;
+}
+
+interface SocialLink {
+  href: string;
+  icon: IconType;
+  lightHoverColor: string;
+  darkHoverColor: string;
+}
+
+const StyledIcon: React.FC<StyledIconProps> = ({ icon: Icon, className, size, color, ...props }) => {
+  return (
+    <span className={`inline-block ${className}`} {...props}>
+      <Icon size={size} color={color} />
+    </span>
+  );
+};
+
+// Social links configuration moved outside the component
+const socialLinks: SocialLink[] = [
+  {
+    href: "",
+    icon: FaGithub,
+    lightHoverColor: "hover:text-gray-900",
+    darkHoverColor: "dark:hover:text-gray-200",
+  },
+  {
+    href: "",
+    icon: FaTwitter,
+    lightHoverColor: "hover:text-blue-400",
+    darkHoverColor: "dark:hover:text-blue-300",
+  },
+  {
+    href: "",
+    icon: FaLinkedin,
+    lightHoverColor: "hover:text-blue-600",
+    darkHoverColor: "dark:hover:text-blue-400",
+  },
+  {
+    href: "",
+    icon: FaEnvelope,
+    lightHoverColor: "hover:text-red-500",
+    darkHoverColor: "dark:hover:text-red-400",
+  },
+];
+
+const SimzeProfileCard: React.FC = () => {
```

Comment:
Since it serves as a page in your Next.js application, you might also want to use the NextPage type. It extends React.FC and includes additional properties and methods that are specific to Next.js, such as getInitialProps.

Reply:
updated